from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_NotePad(object):
    def setupUi(self, NotePad):
        NotePad.setObjectName("NotePad")
        NotePad.resize(868, 814)
        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap(":/img/resources/_app_icon.ico"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        NotePad.setWindowIcon(icon)
        self.centralwidget = QtWidgets.QWidget(NotePad)
        self.centralwidget.setObjectName("centralwidget")
        self.verticalLayout = QtWidgets.QVBoxLayout(self.centralwidget)
        self.verticalLayout.setObjectName("verticalLayout")
        self.textEdit = QtWidgets.QTextEdit(self.centralwidget)
        font = QtGui.QFont()
        font.setFamily("Calibri")
        font.setPointSize(12)
        self.textEdit.setFont(font)
        self.textEdit.setObjectName("textEdit")
        self.verticalLayout.addWidget(self.textEdit)
        NotePad.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(NotePad)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 868, 22))
        self.menubar.setObjectName("menubar")
        self.menuFile = QtWidgets.QMenu(self.menubar)
        self.menuFile.setObjectName("menuFile")
        self.menuEdit = QtWidgets.QMenu(self.menubar)
        self.menuEdit.setObjectName("menuEdit")
        self.menuChange = QtWidgets.QMenu(self.menubar)
        self.menuChange.setObjectName("menuChange")
        self.menuInsert = QtWidgets.QMenu(self.menubar)
        self.menuInsert.setObjectName("menuInsert")
        self.menuHelp = QtWidgets.QMenu(self.menubar)
        self.menuHelp.setObjectName("menuHelp")
        NotePad.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(NotePad)
        self.statusbar.setObjectName("statusbar")
        NotePad.setStatusBar(self.statusbar)
        self.toolBar = QtWidgets.QToolBar(NotePad)
        self.toolBar.setMovable(False)
        self.toolBar.setObjectName("toolBar")
        NotePad.addToolBar(QtCore.Qt.TopToolBarArea, self.toolBar)
        self.actionNew = QtWidgets.QAction(NotePad)
        icon1 = QtGui.QIcon()
        icon1.addPixmap(QtGui.QPixmap(":/img/resources/new.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionNew.setIcon(icon1)
        self.actionNew.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionNew.setObjectName("actionNew")
        self.actionOpen = QtWidgets.QAction(NotePad)
        icon2 = QtGui.QIcon()
        icon2.addPixmap(QtGui.QPixmap(":/img/resources/open.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionOpen.setIcon(icon2)
        self.actionOpen.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionOpen.setObjectName("actionOpen")
        self.actionSave = QtWidgets.QAction(NotePad)
        icon3 = QtGui.QIcon()
        icon3.addPixmap(QtGui.QPixmap(":/img/resources/save.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionSave.setIcon(icon3)
        self.actionSave.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionSave.setObjectName("actionSave")
        self.actionSave_as = QtWidgets.QAction(NotePad)
        icon4 = QtGui.QIcon()
        icon4.addPixmap(QtGui.QPixmap(":/img/resources/save_as.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionSave_as.setIcon(icon4)
        self.actionSave_as.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionSave_as.setObjectName("actionSave_as")
        self.actionCut = QtWidgets.QAction(NotePad)
        icon5 = QtGui.QIcon()
        icon5.addPixmap(QtGui.QPixmap(":/img/resources/Cut.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionCut.setIcon(icon5)
        self.actionCut.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionCut.setObjectName("actionCut")
        self.actionCopy = QtWidgets.QAction(NotePad)
        icon6 = QtGui.QIcon()
        icon6.addPixmap(QtGui.QPixmap(":/img/resources/Copy.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionCopy.setIcon(icon6)
        self.actionCopy.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionCopy.setObjectName("actionCopy")
        self.actionPaste = QtWidgets.QAction(NotePad)
        icon7 = QtGui.QIcon()
        icon7.addPixmap(QtGui.QPixmap(":/img/resources/Paste.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionPaste.setIcon(icon7)
        self.actionPaste.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionPaste.setObjectName("actionPaste")
        self.actionAlign_Left = QtWidgets.QAction(NotePad)
        icon8 = QtGui.QIcon()
        icon8.addPixmap(QtGui.QPixmap(":/img/resources/text_align_left.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionAlign_Left.setIcon(icon8)
        self.actionAlign_Left.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionAlign_Left.setObjectName("actionAlign_Left")
        self.actionAlign_Right = QtWidgets.QAction(NotePad)
        icon9 = QtGui.QIcon()
        icon9.addPixmap(QtGui.QPixmap(":/img/resources/text_align_right.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionAlign_Right.setIcon(icon9)
        self.actionAlign_Right.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionAlign_Right.setObjectName("actionAlign_Right")
        self.actionAlign_Center = QtWidgets.QAction(NotePad)
        icon10 = QtGui.QIcon()
        icon10.addPixmap(QtGui.QPixmap(":/img/resources/text_align_center.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionAlign_Center.setIcon(icon10)
        self.actionAlign_Center.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionAlign_Center.setObjectName("actionAlign_Center")
        self.actionAlign_Justify = QtWidgets.QAction(NotePad)
        icon11 = QtGui.QIcon()
        icon11.addPixmap(QtGui.QPixmap(":/img/resources/text_align_justify.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionAlign_Justify.setIcon(icon11)
        self.actionAlign_Justify.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionAlign_Justify.setObjectName("actionAlign_Justify")
        self.actionBold = QtWidgets.QAction(NotePad)
        icon12 = QtGui.QIcon()
        icon12.addPixmap(QtGui.QPixmap(":/img/resources/edit-bold.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionBold.setIcon(icon12)
        self.actionBold.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionBold.setObjectName("actionBold")
        self.actionItalic = QtWidgets.QAction(NotePad)
        icon13 = QtGui.QIcon()
        icon13.addPixmap(QtGui.QPixmap(":/img/resources/text_italic.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionItalic.setIcon(icon13)
        self.actionItalic.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionItalic.setObjectName("actionItalic")
        self.actionUnderline = QtWidgets.QAction(NotePad)
        icon14 = QtGui.QIcon()
        icon14.addPixmap(QtGui.QPixmap(":/img/resources/underline_icon.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionUnderline.setIcon(icon14)
        self.actionUnderline.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionUnderline.setObjectName("actionUnderline")
        self.actionUndo = QtWidgets.QAction(NotePad)
        icon15 = QtGui.QIcon()
        icon15.addPixmap(QtGui.QPixmap(":/img/resources/Undo.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionUndo.setIcon(icon15)
        self.actionUndo.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionUndo.setObjectName("actionUndo")
        self.actionRedo = QtWidgets.QAction(NotePad)
        icon16 = QtGui.QIcon()
        icon16.addPixmap(QtGui.QPixmap(":/img/resources/Redo.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionRedo.setIcon(icon16)
        self.actionRedo.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionRedo.setObjectName("actionRedo")
        self.actionFont = QtWidgets.QAction(NotePad)
        icon17 = QtGui.QIcon()
        icon17.addPixmap(QtGui.QPixmap(":/img/resources/font.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionFont.setIcon(icon17)
        self.actionFont.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionFont.setObjectName("actionFont")
        self.actionFont_Color = QtWidgets.QAction(NotePad)
        icon18 = QtGui.QIcon()
        icon18.addPixmap(QtGui.QPixmap(":/img/resources/change_text_color.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionFont_Color.setIcon(icon18)
        self.actionFont_Color.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionFont_Color.setObjectName("actionFont_Color")
        self.actionBackground_Color = QtWidgets.QAction(NotePad)
        icon19 = QtGui.QIcon()
        icon19.addPixmap(QtGui.QPixmap(":/img/resources/bg_color.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionBackground_Color.setIcon(icon19)
        self.actionBackground_Color.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionBackground_Color.setObjectName("actionBackground_Color")
        self.actionInsert_Date = QtWidgets.QAction(NotePad)
        icon20 = QtGui.QIcon()
        icon20.addPixmap(QtGui.QPixmap(":/img/resources/Calendar.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionInsert_Date.setIcon(icon20)
        self.actionInsert_Date.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionInsert_Date.setObjectName("actionInsert_Date")
        self.actionInsert_Time = QtWidgets.QAction(NotePad)
        icon21 = QtGui.QIcon()
        icon21.addPixmap(QtGui.QPixmap(":/img/resources/Clock.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionInsert_Time.setIcon(icon21)
        self.actionInsert_Time.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionInsert_Time.setObjectName("actionInsert_Time")
        self.actionExport_to_PDF = QtWidgets.QAction(NotePad)
        icon22 = QtGui.QIcon()
        icon22.addPixmap(QtGui.QPixmap(":/img/resources/doc_pdf.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionExport_to_PDF.setIcon(icon22)
        self.actionExport_to_PDF.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionExport_to_PDF.setObjectName("actionExport_to_PDF")
        self.actionPrint_Preview = QtWidgets.QAction(NotePad)
        icon23 = QtGui.QIcon()
        icon23.addPixmap(QtGui.QPixmap(":/img/resources/Preview.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionPrint_Preview.setIcon(icon23)
        self.actionPrint_Preview.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionPrint_Preview.setObjectName("actionPrint_Preview")
        self.actionPrint = QtWidgets.QAction(NotePad)
        icon24 = QtGui.QIcon()
        icon24.addPixmap(QtGui.QPixmap(":/img/resources/Print.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionPrint.setIcon(icon24)
        self.actionPrint.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionPrint.setObjectName("actionPrint")
        self.actionSelect_All = QtWidgets.QAction(NotePad)
        icon25 = QtGui.QIcon()
        icon25.addPixmap(QtGui.QPixmap(":/img/resources/select_all.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionSelect_All.setIcon(icon25)
        self.actionSelect_All.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionSelect_All.setObjectName("actionSelect_All")
        self.actionClear_All = QtWidgets.QAction(NotePad)
        icon26 = QtGui.QIcon()
        icon26.addPixmap(QtGui.QPixmap(":/img/resources/clear_all.ico"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionClear_All.setIcon(icon26)
        self.actionClear_All.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionClear_All.setObjectName("actionClear_All")
        self.actionAbout = QtWidgets.QAction(NotePad)
        icon27 = QtGui.QIcon()
        icon27.addPixmap(QtGui.QPixmap(":/img/resources/app_info.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionAbout.setIcon(icon27)
        self.actionAbout.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionAbout.setObjectName("actionAbout")
        self.actionExit = QtWidgets.QAction(NotePad)
        icon28 = QtGui.QIcon()
        icon28.addPixmap(QtGui.QPixmap(":/img/resources/exit.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionExit.setIcon(icon28)
        self.actionExit.setMenuRole(QtWidgets.QAction.NoRole)
        self.actionExit.setObjectName("actionExit")
        self.menuFile.addAction(self.actionNew)
        self.menuFile.addAction(self.actionOpen)
        self.menuFile.addAction(self.actionSave)
        self.menuFile.addAction(self.actionSave_as)
        self.menuFile.addSeparator()
        self.menuFile.addAction(self.actionPrint)
        self.menuFile.addAction(self.actionPrint_Preview)
        self.menuFile.addAction(self.actionExport_to_PDF)
        self.menuFile.addSeparator()
        self.menuFile.addAction(self.actionExit)
        self.menuEdit.addAction(self.actionCopy)
        self.menuEdit.addAction(self.actionCut)
        self.menuEdit.addAction(self.actionPaste)
        self.menuEdit.addSeparator()
        self.menuEdit.addAction(self.actionUndo)
        self.menuEdit.addAction(self.actionRedo)
        self.menuEdit.addSeparator()
        self.menuEdit.addAction(self.actionSelect_All)
        self.menuChange.addAction(self.actionFont_Color)
        self.menuChange.addAction(self.actionBackground_Color)
        self.menuInsert.addAction(self.actionInsert_Date)
        self.menuInsert.addAction(self.actionInsert_Time)
        self.menuHelp.addAction(self.actionAbout)
        self.menubar.addAction(self.menuFile.menuAction())
        self.menubar.addAction(self.menuEdit.menuAction())
        self.menubar.addAction(self.menuChange.menuAction())
        self.menubar.addAction(self.menuInsert.menuAction())
        self.menubar.addAction(self.menuHelp.menuAction())
        self.toolBar.addAction(self.actionNew)
        self.toolBar.addAction(self.actionOpen)
        self.toolBar.addAction(self.actionSave)
        self.toolBar.addSeparator()
        self.toolBar.addAction(self.actionCut)
        self.toolBar.addAction(self.actionCopy)
        self.toolBar.addAction(self.actionPaste)
        self.toolBar.addSeparator()
        self.toolBar.addAction(self.actionBold)
        self.toolBar.addAction(self.actionItalic)
        self.toolBar.addAction(self.actionUnderline)
        self.toolBar.addSeparator()
        self.toolBar.addAction(self.actionAlign_Left)
        self.toolBar.addAction(self.actionAlign_Center)
        self.toolBar.addAction(self.actionAlign_Right)
        self.toolBar.addAction(self.actionAlign_Justify)
        self.toolBar.addSeparator()
        
        # Font Bar
        self.toolBar.addSeparator()
        self.fontcombo = QtWidgets.QFontComboBox()
        self.fontcombo.setCurrentFont(font)
        self.toolBar.addWidget(self.fontcombo)

        self.toolBar.addSeparator()

        self.font_size = QtWidgets.QSpinBox()
        self.font_size.setValue(12)
        self.toolBar.addWidget(self.font_size)

        self.toolBar.addSeparator()
        # ================================================
        
        self.toolBar.addAction(self.actionUndo)
        self.toolBar.addAction(self.actionRedo)
        self.toolBar.addSeparator()
        self.toolBar.addAction(self.actionFont)
        self.toolBar.addAction(self.actionFont_Color)
        self.toolBar.addAction(self.actionBackground_Color)
        self.toolBar.addSeparator()
        self.toolBar.addAction(self.actionExport_to_PDF)
        self.toolBar.addAction(self.actionPrint_Preview)
        self.toolBar.addAction(self.actionPrint)
        self.toolBar.addSeparator()
        self.toolBar.addAction(self.actionInsert_Date)
        self.toolBar.addAction(self.actionInsert_Time)

        self.retranslateUi(NotePad)
        QtCore.QMetaObject.connectSlotsByName(NotePad)

    def retranslateUi(self, NotePad):
        _translate = QtCore.QCoreApplication.translate
        NotePad.setWindowTitle(_translate("NotePad", "NotePad"))
        self.menuFile.setTitle(_translate("NotePad", "File"))
        self.menuEdit.setTitle(_translate("NotePad", "Edit"))
        self.menuChange.setTitle(_translate("NotePad", "Change"))
        self.menuInsert.setTitle(_translate("NotePad", "Insert"))
        self.menuHelp.setTitle(_translate("NotePad", "Help"))
        self.toolBar.setWindowTitle(_translate("NotePad", "toolBar"))
        self.actionNew.setText(_translate("NotePad", "New"))
        self.actionNew.setToolTip(_translate("NotePad", "Create a new file"))
        self.actionNew.setShortcut(_translate("NotePad", "Ctrl+N"))
        self.actionOpen.setText(_translate("NotePad", "Open"))
        self.actionOpen.setToolTip(_translate("NotePad", "Open a file"))
        self.actionOpen.setShortcut(_translate("NotePad", "Ctrl+O"))
        self.actionSave.setText(_translate("NotePad", "Save"))
        self.actionSave.setToolTip(_translate("NotePad", "Save this file"))
        self.actionSave.setShortcut(_translate("NotePad", "Ctrl+S"))
        self.actionSave_as.setText(_translate("NotePad", "Save As"))
        self.actionSave_as.setToolTip(_translate("NotePad", "Save as..."))
        self.actionSave_as.setShortcut(_translate("NotePad", "Ctrl+Shift+S"))
        self.actionCut.setText(_translate("NotePad", "Cut"))
        self.actionCut.setToolTip(_translate("NotePad", "Cut"))
        self.actionCut.setShortcut(_translate("NotePad", "Ctrl+X"))
        self.actionCopy.setText(_translate("NotePad", "Copy"))
        self.actionCopy.setToolTip(_translate("NotePad", "Copy selected text"))
        self.actionCopy.setShortcut(_translate("NotePad", "Ctrl+C"))
        self.actionPaste.setText(_translate("NotePad", "Paste"))
        self.actionPaste.setToolTip(_translate("NotePad", "Paste text"))
        self.actionPaste.setShortcut(_translate("NotePad", "Ctrl+V"))
        self.actionAlign_Left.setText(_translate("NotePad", "Align Left"))
        self.actionAlign_Left.setToolTip(_translate("NotePad", "Align text to the left"))
        self.actionAlign_Left.setShortcut(_translate("NotePad", "Ctrl+L"))
        self.actionAlign_Right.setText(_translate("NotePad", "Align Right"))
        self.actionAlign_Right.setToolTip(_translate("NotePad", "Align text to the right"))
        self.actionAlign_Right.setShortcut(_translate("NotePad", "Ctrl+R"))
        self.actionAlign_Center.setText(_translate("NotePad", "Align Center"))
        self.actionAlign_Center.setToolTip(_translate("NotePad", "Align text to the center"))
        self.actionAlign_Center.setShortcut(_translate("NotePad", "Ctrl+E"))
        self.actionAlign_Justify.setText(_translate("NotePad", "Align Justify"))
        self.actionAlign_Justify.setToolTip(_translate("NotePad", "Justify text alignment"))
        self.actionAlign_Justify.setShortcut(_translate("NotePad", "Ctrl+J"))
        self.actionBold.setText(_translate("NotePad", "Bold"))
        self.actionBold.setToolTip(_translate("NotePad", "Make text bold"))
        self.actionBold.setShortcut(_translate("NotePad", "Ctrl+B"))
        self.actionItalic.setText(_translate("NotePad", "Italic"))
        self.actionItalic.setToolTip(_translate("NotePad", "Italicize text"))
        self.actionItalic.setShortcut(_translate("NotePad", "Ctrl+I"))
        self.actionUnderline.setText(_translate("NotePad", "Underline"))
        self.actionUnderline.setToolTip(_translate("NotePad", "Underline text"))
        self.actionUndo.setText(_translate("NotePad", "Undo"))
        self.actionUndo.setToolTip(_translate("NotePad", "Undo recent activity"))
        self.actionUndo.setShortcut(_translate("NotePad", "Ctrl+Z"))
        self.actionRedo.setText(_translate("NotePad", "Redo"))
        self.actionRedo.setToolTip(_translate("NotePad", "Redo recent activity"))
        self.actionRedo.setShortcut(_translate("NotePad", "Ctrl+Y"))
        self.actionFont.setText(_translate("NotePad", "Font"))
        self.actionFont.setToolTip(_translate("NotePad", "Change font options"))
        self.actionFont_Color.setText(_translate("NotePad", "Font Color"))
        self.actionFont_Color.setToolTip(_translate("NotePad", "Change font color"))
        self.actionBackground_Color.setText(_translate("NotePad", "Background Color"))
        self.actionBackground_Color.setToolTip(_translate("NotePad", "Change background color"))
        self.actionInsert_Date.setText(_translate("NotePad", "Insert Date"))
        self.actionInsert_Date.setToolTip(_translate("NotePad", "Insert today\'s date"))
        self.actionInsert_Time.setText(_translate("NotePad", "Insert Time"))
        self.actionInsert_Time.setToolTip(_translate("NotePad", "Insert current time"))
        self.actionExport_to_PDF.setText(_translate("NotePad", "Export to PDF"))
        self.actionExport_to_PDF.setToolTip(_translate("NotePad", "Export document as PDF"))
        self.actionPrint_Preview.setText(_translate("NotePad", "Print Preview"))
        self.actionPrint_Preview.setToolTip(_translate("NotePad", "Open print preview"))
        self.actionPrint.setText(_translate("NotePad", "Print"))
        self.actionPrint.setToolTip(_translate("NotePad", "Print document"))
        self.actionPrint.setShortcut(_translate("NotePad", "Ctrl+P"))
        self.actionSelect_All.setText(_translate("NotePad", "Select All"))
        self.actionSelect_All.setToolTip(_translate("NotePad", "Select all text"))
        self.actionSelect_All.setShortcut(_translate("NotePad", "Ctrl+A"))
        self.actionClear_All.setText(_translate("NotePad", "Clear All"))
        self.actionClear_All.setToolTip(_translate("NotePad", "Clear all text"))
        self.actionClear_All.setShortcut(_translate("NotePad", "Shift+Backspace"))
        self.actionAbout.setText(_translate("NotePad", "About"))
        self.actionAbout.setToolTip(_translate("NotePad", "About app"))
        self.actionExit.setText(_translate("NotePad", "Exit"))
        self.actionExit.setToolTip(_translate("NotePad", "Quit program"))
        self.actionExit.setShortcut(_translate("NotePad", "Ctrl+Q"))
import res_rc


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    NotePad = QtWidgets.QMainWindow()
    ui = Ui_NotePad()
    ui.setupUi(NotePad)
    NotePad.show()
    sys.exit(app.exec_())
